Tasks/invert.js

```diff
@@ -1,14 +1,14 @@
 // Reverse an array, you can't use .reverse()
 'use strict';
 
-const invert = arr => {
+const invert = (arr) => {
   const arrLength = arr.length;
   const invertedArr = new Array(arrLength);
-  
+
   let newIndex = arrLength - 1;
```

Comment:
In my opinion, there might be better variants of how to name indexes `newIndex` and `index`.
There some names come to my mind
1. `left` and `right`
2. `leftIndex` and `rightIndex`
3. `indexArr` and `indexInvArr`

Reply:
I think that the names `indexArr `and `indexInvArr `will be the best decision